Uno.Gallery/Uno.Gallery.UWP/App.xaml

```diff
@@ -11,6 +11,8 @@
 				<!-- Load WinUI resources -->
 				<XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
+                <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
```

Comment:
Format document please ;)
![image](https://user-images.githubusercontent.com/16295702/143943422-7069ce19-9bf6-4da4-9622-f432c10f7d57.png)

Reply:
yes, generally tab is preferred here. but the indentation level is fine.

Reply:
@agneszitte-nventive you need https://chrome.google.com/webstore/detail/tab-size-on-github/ofjbgncegkdemndciafljngjbdpfmbkn